pkg/repositories/transformers/execution_test.go

```diff
@@ -481,15 +488,24 @@ func TestFromExecutionModel(t *testing.T) {
 	specBytes, _ := proto.Marshal(spec)
 	phase := core.WorkflowExecution_RUNNING.String()
 	startedAt := time.Date(2018, 8, 30, 0, 0, 0, 0, time.UTC)
+	createdAt := time.Date(2022, 01, 18, 0, 0, 0, 0, time.UTC)
 	startedAtProto, _ := ptypes.TimestampProto(startedAt)
+	createdAtProto, _ := ptypes.TimestampProto(createdAt)
 	closure := admin.ExecutionClosure{
 		ComputedInputs: spec.Inputs,
 		Phase:          core.WorkflowExecution_RUNNING,
 		StartedAt:      startedAtProto,
+		StateChangeDetails: &admin.ExecutionStateChangeDetails{
+			State:      admin.ExecutionState_EXECUTION_ACTIVE,
+			OccurredAt: createdAtProto,
+		},
 	}
 	closureBytes, _ := proto.Marshal(&closure)
 
 	executionModel := models.Execution{
+		BaseModel: models.BaseModel{
```

Comment:
can you add the expected state column values to the execution model too?

Reply:
Done